src/controllers/controller.js

```diff
@@ -133,7 +133,7 @@ class Controller {
 			geoCache.save(true)
 			return result
 		} catch (err) {
-			throw { source: 'getAddress', error: err }
+			return { addr: 'Unknown', flag: '' }
```

Comment:
Going through with notification is very good idea but completely silencing error is not.
How about adding this to error log?
`this.log.error('getAddress: failed to fetch data', err)`

Reply:
I was thinking about the permanent 'not going to install nominatim' situation and log bloat - but you're right this should perhaps be through a config.  I will add a log and do something different for a 'none' situation

Reply:
Working like a charm, thank you!